GDAXSharp/WebSocket/Models/Request/TickerChannel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace GDAXSharp.WebSocket.Models.Request
+{
+    public class TickerChannel
+    {
+        [JsonProperty("type")]
+        public string Type { get; set; }
```

Comment:
you can convert these into their enums doing something like this `[JsonConverter(typeof(StringEnumConverter))]`

here is another example https://github.com/dougdellolio/gdax-csharp/blob/master/GDAXSharp/Services/Orders/Models/Responses/OrderResponse.cs